TerrorConsole/Assets/_Main/Scripts/Objects/Doors/Door.cs

```diff
@@ -7,11 +7,25 @@ public class Door : MonoBehaviour
     {
         [SerializeField] protected bool _isLocked;
         [SerializeField] private SpriteRenderer _spriteRenderer;
-        [SerializeField] private Sprite newSprite;
+        [SerializeField] private Sprite openSprite;
+        [SerializeField] private Sprite closedSprite;
         [SerializeField] private Collider2D _collider2D;
+        [SerializeField] private LevelEventsRecorder _eventRecorder;
         [SerializeField] private UnityEvent _onDoorOpened = new UnityEvent();
         [SerializeField] private UnityEvent _onDoorClosed = new UnityEvent();
 
+        private void Start()
+        {
+            if (_eventRecorder.CheckEventState())
+            {
+                OpenDoor();
+            }
+            else
+            {
+                CloseDoor();
```

Comment:
This will eventually trigger sounds, use optional parameters to allow an instant change and avoid animations or sounds.

Like `private void OpenDoor(bool instantly = false)`